docs/vgclog_examples.md

```diff
@@ -0,0 +1,1448 @@
+  <response-info timems="3.138" idlems="1.056" threads="2" lastid="00000000006EDE00" lastname="RunDataWriter.1" />
+</exclusive-start>
+```
+
+A `balanced` partial GC cycle is triggered when the region count for the *eden* space reaches the allocation taxation threshold. The logs record this trigger reason by using the`<allocation-taxation>` element.
+
+```xml
+<allocation-taxation id="185" taxation-threshold="2147483648" timestamp="2021-02-26T11:11:42.311" intervalms="3745.785" />
+```
+
+Details about the start of the cycle are recorded by the `<cycle-start>` element. The cycle is recorded as a `partial gc` with an `id=336`. Any subsequent elements associated with this cycle have a `contextid=186` to match the cycle's `id`. You can use this `contextid` value to distinguish the partial GC cycle increment and operations from interleaving increments and operations from other `balanced` cycles, such as global *mark* cycles.
+
+```xml
+<cycle-start id="186" type="partial gc" contextid="0" timestamp="2021-02-26T11:11:42.311" intervalms="3745.805" />
+
+```
+
+The partial cycle begins its one and only GC increment, recorded using the `<gc-start>` element. You can understand the effect that the increment operations have on the heap by comparing snapshots of the memory taken at the start and the end of the increment.  The child elements `<mem-info>` and `<mem>` of the `<gc-start>` and `<gc-end>` elements record the amount of memory available and where it is located in the heap.
+
+```xml
+<gc-start id="187" type="partial gc" contextid="186" timestamp="2021-02-26T11:11:42.311">
+  <mem-info id="188" free="897581056" total="4294967296" percent="20">
+    <mem type="eden" free="0" total="2147483648" percent="0" />
+    <arraylet-primitive objects="1" leaves="4" largest="4" />
+    <remembered-set count="2749664" freebytes="160705664" totalbytes="171704320" percent="93" regionsoverflowed="1" regionsstable="12" regionsrebuilding="0"/>
+  </mem-info>
+</gc-start>
+```
+
+As expected, at the start of this increment, the eden region is full. 856MB (897,581,056 B) of the total 4096MB (4294,967,296 B) heap, equivalent to 20% of the heap, is available as free memory.
+
+The status of the *remembered set*, a metastructure specific to openJ9 JVM generational garbage collectors, is reported by the `<remembered-set>` element. The remembered set metastructure keeps track of any object references that cross different regions. Each region corresponds to a single remembered set. 
+
+The partial gc cycle uses and prunes the remembered set. The `regionsoverflowed` value records the number of regions that exceeded the non-object-heap memory allocation that is reserved for the remembered set. The partial GC cycle cannot reclaim memory from these overflow regions. The partial GC cycle also cannot reclaim memory from any regions whose remembered set is being rebuilt by an increment of a global mark cycle that is in progress.
```

Comment:
I need definitions of collection set, vs. remembered set, vs. 'roots'. Ask Alexandar.

Not really anything on collection sets in the gc topic on balanced.

Reply:
I think I defined 'collection set' mean while, but anyhow it's again: it's a (sub)set of regions that are being garbage collected during Partial GC. 

Anyhow, my comment here was about going into more details, since 'use' and 'prune' does not refer to same regions. 
We use RSs of regions that we GC (from collection as). We used them as roots (to find initial object to mark and scan). We prune RSs of regions the we don't not GC (outside of collection set).